package.json

```diff
@@ -90,7 +90,6 @@
     "@types/express": "^4.17.1",
     "@types/jest": "^24.0.20",
     "@types/luxon": "^1.21.0",
-    "@types/node": "^11.15.0",
```

Comment:
Why was this removed? I believe it's still required

Reply:
It was removed while I was trying to installing typeorm. I will install the `@types/node` and fix it!

Reply:
@johnkahn do we want to keep version "11.15.0"? I installed it and it is showing me version "14.11.2"

Reply:
@johnkahn it was moved to `dependencies`

@smitshah2531 unless there was a reason for the version bump, let's handle that in a separate issue. Just revert this change back to the original.